pkg/v13/resource/tiller/resource.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/giantswarm/tenantcluster"
 
 	"github.com/giantswarm/cluster-operator/pkg/cluster"
-	"github.com/giantswarm/cluster-operator/pkg/v10/key"
+	"github.com/giantswarm/cluster-operator/pkg/v13/key"
```

Comment:
Okay, we always link it this to `v10`, how we did not get this when we have new versions?

Reply:
Good catch. New version bundles should always be added using the bash magic. 

https://github.com/giantswarm/giantswarm/blob/master/processes/new-version-bundle.md

But the multiple controller structure is tricky and sometimes it goes wrong.

The whole multiple controllers design is a PITA IMO. With the new clusterapi controller we will get rid of it. But using it for all providers will take a long time.